testdata/kata/OCP-38468/pod_with_init_container.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: myapp-container
+    image: busybox:1.28
```

Comment:
Better to use image from quay as there are limits from docker

Reply:
@liangxia updated